apps/client/src/features/modals/ViewsSettingsModal.jsx

```diff
@@ -75,74 +82,109 @@ export default function ViewsSettingsModal() {
+            Read the docs
           </a>
+        </p>
+        <div className={style.hSeparator}>Style Options</div>
+        <div className={styles.splitSection}>
+          <div>
+            <span className={`${styles.sectionTitle} ${styles.main}`}>Override CSS Styles</span>
+            <span className={styles.sectionSubtitle}>Enable / Disable override</span>
+          </div>
+          <Switch onChange={() => handleChange('overrideStyles', !formData.overrideStyles)} variant='ontime-on-light' />
         </div>
-        <div className={style.modalFields}>
-          <div className={style.modalInline}>
-            <FormControl>
-              <FormLabel htmlFor='overrideStyles'>
-                Override CSS Styles
-                <span className={style.labelNote}>
-                  <br />
-                  Enable / Disable override
-                </span>
-              </FormLabel>
-              <EnableBtn
-                active={formData.overrideStyles}
-                text={formData.overrideStyles ? 'Style Override Enabled' : 'Style Override Disabled'}
-                actionHandler={() => handleChange('overrideStyles', !formData.overrideStyles)}
-              />
-            </FormControl>
+        <hr className={styles.divider} />
+        <div className={style.hSeparator}>Timer Options</div>
+        <div className={styles.splitSection}>
+          <div>
+            <span className={`${styles.sectionTitle} ${styles.main}`}>Normal Color</span>
+            <span className={styles.sectionSubtitle}>Change timer Normal Color</span>
+          </div>
+          <PopoverPicker
+            name='normalColor'
+            color={formData.normalColor}
+            onChange={(event) => handleChange('normalColor', event)}
+          />
+        </div>
+        <div className={styles.splitSection}>
+          <div>
+            <span className={`${styles.sectionTitle} ${styles.main}`}>Warning Color</span>
+            <span className={styles.sectionSubtitle}>Change timer Warning Color</span>
           </div>
+          <PopoverPicker
+            name='warningColor'
+            color={formData.warningColor}
+            onChange={(event) => handleChange('warningColor', event)}
+          />
+        </div>
+        <FormControl className={styles.splitSection}>
+          <label htmlFor='warningThreshold'>
+            <span className={styles.sectionTitle}>Warning Time</span>
+            <span className={styles.sectionSubtitle}>The time (in minutes) when the color changes</span>
+          </label>
+          <InputGroup size='sm' width='140px'>
+            <NumberInput
+              {...numberInputProps}
+              id='warningThreshold'
+              variant='ontime-filled-on-light'
+              value={millisToMinutes(Number(formData.warningThreshold), 'm')}
+              onChange={(event) => handleThresholdChange('warningThreshold', event)}
+            >
+              <NumberInputField />
+            </NumberInput>
```

Comment:
You should try to do some validation here, see the OSCSettings form for some inspiration

I usually prefer always working with strings